livekitlib

```diff
@@ -247,7 +247,7 @@ device_bestfs()
    local FS
 
    FS="$(blkid "$1" | sed -r "s/.*TYPE=//" | tr -d '"' | tr [A-Z] [a-z])"
-   if [ "$FS" = "msdos" -o "$FS" = "fat" -o "$FS" = "vfat" ]; then
+   if [ "$FS" = "msdos" -o "$FS" = "fat" ]; then
```

Comment:
Because if "$FS" were to be "vfat", the following line \(FS=vfat) would be a no-op.